### PR TITLE
Ropes Issue #13 - Required Tags

### LINKED ISSUE #13: Required Tag Association
#### As a user I want all images to have at least one tag.

__Story__:

The tag feature has been a hit. Our users love being able to filter images by
cats, SR-71s, and "AppFolio tshirts" to name a few. In fact, we have discovered
that images without any tags receive significantly less views than those with
tags. As a result, we have decided to require all images to have at least one
tag.

__Acceptance criteria__:
- [x] I cannot save an image without adding at least one tag.
- [x] An error message is associated with the appropriate input field when a
tag is not provided.
- [x] All images have at least one tag.

__Discussion Topic__:
- [ ] What do you do about existing images that have no tags?

__Dependencies__:
- Image Tags
- Refactor flow tests to use page objects

### PR DESCRIPTION
**Pull request by zldavis**
Closes #13

### CODE REVIEW

File: app/views/images/new.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <%= simple_form_for @image, url: {action: "create"} do |f| %>
   <%= f.input :url, required: false %>
-  <%= f.input :tag_list, input_html: {value: @image.tag_list.join(', ')} %>
+  <%= f.input :tag_list, required: false, input_html: {value: @image.tag_list.join(', ')} %>
```
Comment (pololee) [inline app/views/images/new.html.erb]: I am confused here. The `tag_list` is a required input in this PR.
Shouldn't it be `required: true`?

Comment (zldavis) [inline app/views/images/new.html.erb]: screen shot 2017-06-16 at 11 50 39 am

Its just a simple_form option. By default simple_form will add this funky element and required asterix onto a label for a field it deems required. Earlier on in the project talking with Aaron on the style he said to just use the required:false option on these fields to get rid of that look.

Comment (pololee) [inline app/views/images/new.html.erb]: Oh. I see. It's a kind of a convention in our app that having `*` on the label indicates this is a required field. But I am Ok not to have it. Thanks.

Comment (zldavis) [inline app/views/images/new.html.erb]: oh ok haha, interesting then that Aaron told me to remove it. 🙂

**Pull request merged**